services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie, Genre, Actor
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__movie__in=genres_ids).distinct()
+    if actors_ids:
+        queryset = queryset.filter(actors__movie__in=actors_ids).distinct()
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[Genre] = None,
+        actors_ids: list[Actor] = None
+) -> Movie:
+    create_movie = Movie.objects.create(
+        title=movie_title,
+        description=movie_description
+    )
+    if genres_ids:
+        create_movie.genres.set(genres_ids)
+
+    if actors_ids:
+        create_movie.genres.set(actors_ids)
```

Comment:
There is a mistake here. You should set `create_movie.actors.set(actors_ids)` instead of `create_movie.genres.set(actors_ids)` to associate actors with the movie.